README.md

```diff
@@ -224,6 +203,31 @@ dotnet script -d foo.csx -- -d
 ```
 
 will pass the `-d` before `--` to `dotnet script` and enable the debug mode whereas the `-d` after `--` is passed to script for its own interpretation of the argument.
+
+## Extras
+
+Beyond the standard scripting dialect support (compatible with `csi.exe`), `Dotnet.Script` provides some extra features, located in the `Dotnet.Script.Extras` package.
```

Comment:
So how are the extras enabled now? That information is missing because `#load` using a URL (and the example in the next sub-section) doesn't work anymore.